ddtrace/llmobs/_evaluators/ragas/faithfulness.py

```diff
@@ -13,24 +65,236 @@ class RagasFaithfulnessEvaluator:
     METRIC_TYPE = "score"
 
     def __init__(self, llmobs_service):
+        """
+        Initialize an evaluator that uses the ragas library to generate a faithfulness score on finished LLM spans.
+
+        Faithfulness measures the factual consistency of an LLM's output against a given context.
+        There are two LLM calls required to generate a faithfulness score - one to generate a set of statements from
+        the answer, and another to measure the faithfulness of those statements against the context using natural
+        language entailment.
+
+        For more information, see https://docs.ragas.io/en/latest/concepts/metrics/faithfulness/
+
+        The `ragas.metrics.faithfulness` instance is used for faithfulness scores. If there is not llm attribute set
+        on this instance, it will be set to the default `llm_factory()` which uses openai.
+
+        :param llmobs_service: An instance of the LLM Observability service used for tracing the evaluation and
+                                submitting evaluation metrics.
+        """
+        self.enabled = True
```

Comment:
Should this be enabled by default?

Reply:
If this init function is called, we can assume the user has enabled faithfulness via an environment variable since the class won't be initialized otherwise. This boolean is just a check to make sure the required dependencies for ragas is installed .. and later when `run_and_submit_evaluation` is called, we return immediately if _self.enabled=False_

I think it's clearer if we rename this to `self.ragas_dependencies_present` so illl do that